timescaledb/getting-started/add-data.md

```diff
@@ -1,53 +1,63 @@
-# 4. Add time-series data
+# Add time-series data
 
-To showcase TimescaleDB and get you familiar with its features, we’ll need some
-sample data to play around with. We’ll use the real-world scenario of climate
-change. Data about the climate in a certain region is time-series data, as it
-describes how the climate in that area changes over time.
+To help get you familiar with TimescaleDB's features, you’ll need some
+sample data to use. For this tutorial, we provide real-time 
+stock trade data (aka. "tick data") provided by [twelvedata][twelve-data] for 100 stock symbols for the most recent month. 
 
-## About the dataset
+### About the dataset
```

Comment:
```suggestion
## About the dataset
```

Not skipping heading levels is preferable from a semantics and accessibility perspective